_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -0,0 +1,246 @@
+
+
+Quick Start instructions to install and run Istio in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/){:target="_blank"} (GKE) using [Google Cloud Deployment Manager](https://cloud.google.com/deployment-manager/){:target="_blank"}.
+
+This Quick Start creates a new GKE cluster, installs Istio and then deploys the [BookInfo](https://istio.io/docs/guides/bookinfo.html){:target="_blank"} sample application.   It is essentially
+all the steps detailed in [Istio Kubernetes Intaller](https://istio.io/docs/setup/kubernetes/quick-start.html) combined with Deployment Manager.
+
+> **Note:** The default install will create a GKE [**ALPHA** Cluster](https://cloud.google.com/kubernetes-engine/docs/alpha-clusters){:target="_blank"} that allows for [automatic sidecar injection](https://istio.io/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection){:target="_blank"}.  As it is an Alpha Cluster,
+it does not support automatic node or master upgrades and will only run for 30days.
+
+## Prerequisites
+
+This sample requires a valid Google Cloud Platform project with billing enabled.  If you have not signed already, you could enroll for a $300 US [Free Trial](https://cloud.google.com/free/){:target="_blank"} credit.
+
+You will also need to install and configure [gcloud SDK](https://cloud.google.com/sdk/docs/){:target="_blank"} on your local workstation and include kubectl additional component:
+
+```
+gcloud components install kubectl
+```
+
+Verify gcloud is configured with your login and the project you are using:
+```
+gcloud config list
+```
+
+If you do not wish to install the gcloud client, you can use  [Google Cloud Shell](https://cloud.google.com/shell/docs/){:target="_blank"} to perform most tasks.
+If you choose to use Cloud Shell, you can port forward and proxy using its [Web Preview](https://cloud.google.com/shell/docs/using-web-preview#previewing_the_application){:target="_blank"} feature.  For example, to access Grafana from Cloud Shell, change the kubectl port mapping from 3000:3000 to 8080:3000.  You can simultaneously preview four other consoles via Web Preview proxied on ranges 8080 to 8084.  
+
+> <img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning"
+ style="width: 32px; display:inline" />
+**NOTE:** You must set your default compute service account to include:
+
+> - ```roles/container.admin```  (Kubernetes Engine Admin)
+> - ```Editor```  (on by default)
+
+> To set this, navigate to the IAM section of the [Cloud Console](https://console.cloud.google.com/iam-admin/iam/project){:target="_blank"} and find your default GCE/GKE service account in the following form:
+
+```
+projectNumber-compute@developer.gserviceaccount.com
+```
+
+then under Roles, find the ```Kubernetes Engine``` group and set the role ```Kubernetes Engine Admin```.
+
+
+## Setup
+
+### Launch Deployment Manager
+
+Once you have an account and project enabled, proceed to the following link:
```

Comment:
click the following link to open the Deployment Manager.

Reply:
done

note, i could add a section to point out that you dont' **need** to use the GUI though it's nice...users can just run a gcloud command referencing the template like this

```
gcloud deployment-manager deployments create istio --template https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja --properties gkeClusterName:istio-gke
```

should i add that in?